cli/zoo.py

```diff
@@ -69,10 +69,10 @@ def build():
 @click.argument("auth_key", type=str, default=None)
 @click.argument("port", default=7432, type=int)
 def worker(auth_key, port):
-    from smarts.zoo.worker import listen
+    from smarts.zoo import master
```

Comment:
The industry is moving away from the master/slave terminology, consider using `primary/replica` instead.

Reply:
The term `slave` is not used in the code. The current code uses `master/worker` terminology. 

Do you think `master/worker` terminology is okay or should we use some more descriptive terminology like `controller/worker`, as compared to `primary/replica`?

Reply:
The `master/worker` terminology is now replaced with `manager/worker` terminology.